resources.go

```diff
@@ -1805,7 +1805,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		JavaScript: &tfbridge.JavaScriptInfo{
 			Dependencies: map[string]string{
-				"@pulumi/pulumi":    "^0.16.9",
+				"@pulumi/pulumi":    "dev",
```

Comment:
@ellismg is it ok to reference dev here?

Reply:
We should not check in something referencing dev here. It's fine for doing PRs to get validation, but landing it means we are in a pretty awful state if we ever release this package (in fact, automation now would prevent it from being released).

If I am following along, it should be safe to just do a release of `@pulumi/pulumi` with your most recent changes, right?  If that's the case, we could then update this constraint to be something like ^0.16.11, which is what you want.

If so, we can do the release today (I would like to include https://github.com/pulumi/pulumi/pull/2362 in it)